test/controllers/advanced-search/search.controller.test.ts

```diff
@@ -246,6 +246,20 @@ describe("search.controller.test", () => {
             chai.expect(resp.text).to.contain("Registered on");
             chai.expect(resp.text).to.contain("Removed on");
         });
+
+        it("should not display Incorporation date  as company type is protected cell company", async () => {
```

Comment:
Extra space between `date` and `as`.  :)

Reply:
Cheers, updated